teamsyncd/teamsyncd.cpp

```diff
@@ -34,7 +42,13 @@ int main(int argc, char **argv)
             while (true)
             {
                 Selectable *temps;
-                s.select(&temps);
+                s.select(&temps, 1000); // check every second
+
+                auto diff = duration_cast<seconds>(steady_clock::now() - start_time);
+                if(diff.count() > WR_PENDING_TIME)
+                {
+                    sync.applyState();
```

Comment:
>applyState [](start = 25, length = 10)

Do you need applyState every second after WR_PENDING_TIME? or just once?

Reply:
Refactored to make it more straight.